morango/syncsession.py

```diff
@@ -21,13 +21,23 @@ def _join_with_logical_operator(lst, operator):
     op = ") {operator} (".format(operator=operator)
     return "(({items}))".format(items=op.join(lst))
 
+def _get_server_ip(url):
+    try:
+        return socket.gethostbyname(url)
```

Comment:
I don't think this can take a URL, it needs to be just the host. Should probably pass in `hostname` here instead of url.

```
In [3]: socket.gethostbyname("http://google.com")
---------------------------------------------------------------------------
gaierror                                  Traceback (most recent call last)
/home/jamalex/<ipython-input-3-3f638f07b01c> in <module>()
----> 1 socket.gethostbyname("http://google.com")

gaierror: [Errno -2] Name or service not known

In [4]: socket.gethostbyname("google.com")
Out[4]: '216.58.216.14'
```

Reply:
I actually do just pass in the host, but I should change the param name to reflect that, but it could also be an ip that we are passing in.

Reply:
whoops take that back, I forgot to pass in hostname,